gitops/generate_build.go

```diff
@@ -342,6 +342,9 @@ func getBuildCommonLabelsForComponent(component *appstudiov1alpha1.Component) ma
 func GenerateCommonStorage(component appstudiov1alpha1.Component, name string) *corev1.PersistentVolumeClaim {
 	fsMode := corev1.PersistentVolumeFilesystem
 
+	annotations := getBuildCommonLabelsForComponent(&component)
+	annotations["argocd.argoproj.io/sync-options: Prune"] = "false"
```

Comment:
Key is `argocd.argoproj.io/sync-options` and value is `Prune=false`, so it should be: `annotations["argocd.argoproj.io/sync-options"] = "Prune=false"`